js/src/dom/manipulator.js

```diff
@@ -47,9 +47,13 @@ const Manipulator = {
       ...element.dataset
     }
 
-    Object.keys(attributes).forEach(key => {
-      attributes[key] = normalizeData(attributes[key])
-    })
+    Object.keys(attributes)
+      .filter(key => key.startsWith('bs'))
+      .forEach(key => {
+        let pureKey = key.replace(/^bs/, '')
```

Comment:
since the logic changed here, you should update the according unit tests and add new ones

Reply:
@Johann-S 
Updated 👍

Reply:
I would prefer to just have `bsToggle` in that case

Reply:
@rohit2sharma95 one last request: can you add a couple of test cases with `BS` and `bs` not in the start of the data key name please?

Reply:
Oh, NVM I see you have already covered these cases too :)

Reply:
Yes, these are covered already @XhmikosR. I have not checked for uppercase namespace though.

Reply:
I'm not sure if it makes sense after all, I mean, are those even case sensitive?

Reply:
To filter the namespaced attributes `startsWith` method is used that is already case sensitive. So I think there is no need to check for the uppercase namespace. And I did not check it before because I was thinking that people always write HTML attributes in the small case. 🙂